ha/k8s_setup/ha_setup.py

```diff
@@ -274,6 +284,36 @@ def process(self):
         except Exception as c_err:
             sys.stderr.write(f'HA config command failed: {c_err}.\n')
 
+    def _add_node_health(self) -> None:
+        """
+        Add node health
+        """
+        confStoreAPI = ConftStoreSearch()
```

Comment:
Can we reuse the object of this class created at line no 271 here?

Reply:
sure, will change this.

Reply:
Handled in https://github.com/Seagate/cortx-ha/pull/648. Closing this PR since a new PR has been created.